### DISCUSSION TITLE
Suggestion: Ask consent to boost (and eventually post 'quote toots') from the user who posted the toot

### DISCUSSION THREAD

**Discussion by KJHofstede** [mastodon/mastodon#19832] [General]
I totally understand the policy of 'no quote toots' and I agree with it, but I also notice that people are working around it now by just placing links or screenshots of someone's toot. Because it's not always to dunk on somebody, sometimes it is positive and/or constructive.

What if there was a feature to ask consent first from someone to regularly boost their toot? This feature could have three settings: 'Never boost', 'Ask consent first' and 'Always accept boosts'. This feature would be 'Always accept boosts' by default, because Mastodon thrives on people boosting other people's toots, but I can imagine some people don't want it or want to give consent first. You would have to place a 'boost request' for this person which would be placed on a queue, ready for the other user to approve. Until then, the boost remains hidden.

And eventually, you could introduce then the much wanted 'quote toot' as well as a similar feature. This feature would be on 'Ask consent first' by default, because of the many good arguments against quote tweeting on the bird site. But you could set it to 'Always accept' if you don't care about quote toots, because the people who you interact with ususally use it in a positive way to amplify your toots or add constructive context. Or 'Never quote toot' if you always hate it, without exception.

Lastly, this could also maybe used for placing links. That you can not post a toot with a link to another person's toot, if you don't have consent from the person who posted it. If you do post a toot like this, it will also be hidden until the poster approves.

Because I feel the main problem with 'quote tweets/toots' is not that it's intrinsically evil (there are also positive reasons to 'quote toot'), but it's the lack of consent what makes it evil.

tldr; Introduce a consent based culture around boosting/quote tooting/posting links to toots.

**Comment by laril** [comment]
This is a great idea, and I fully support it! 

My personal experience from *the other social media* is that quote-retweets have been mainly used either to build ideas on top of other posts (often taking a step aside from the origninal discussion, this is why just replying is not feasible) or to introduce and maybe sum up a longer thread (for example introducing a discussion in some other language in my native language). I understand that people have also other kinds of experiences, but the feature itself isn't what causes toxic behavior. I would really love to see 'quote toots' as a feature, as in my experience it is something that supports constructive conversations, and it would be great to have a choice whether to enable them.

**Comment by olivierlacan** [comment]
This is a very interesting suggestion and I would expand it slightly to offer one more setting beyond the ones you suggested: 
- `Never boost`
- `Ask consent first`
- `Always accept boosts`
- 🆕 `Always accept instance-local boosts`

It appears that a lot of drive-by reply animosity on Mastodon arises from boosts breaking (as intended) the barriers of individual instances. While that may be something some Mastodon users welcome, a lot of folks are pelted with replies from random people many hops removed from their home instance who either don't take the time to read & understand a conversation/thread or simply don't *see* prior replies to a thread.

The current makeshift solution to this issue is to limit replies to followers only, but that's a bit drastic. It would be helpful to empower folks to limit boosting when they know a specific post is likely to result in abuse from boost dissemination.

**Comment by pixiekat** [comment]
I feel it would make much more sense to be a global feature but also selectable per toot/post for quote boosts. 

Some posts you might not want to be allowed to be quote boosted but some you might want allowed. 

Also I think adding `Always allow mutuals to quote boost`
